swagger.yml

```
@@ -1,6 +1,6 @@
 swagger: '2.0'
 info:
-  version: 10.0.0
+  version: 10.0.1
```

bei abwärtskompatibler Erweiterung die Minor Version hochsetzen

```suggestion
  version: 10.1.0
```